src/types.ts

```diff
@@ -208,6 +208,13 @@ export interface ScanDirExportsOptions {
    */
   fileFilter?: (file: string) => boolean
 
+  /**
+   * Also register type exports
+   *
+   * @default true
+   */
+  types?: boolean
```

Comment:
This would be enabled by default, and I am thinking of releasing it as a minor (3.4.0). Sounds good to you @danielroe?

Reply:
yes, that does 👍